amethyst_renderer/examples/material.rs

```diff
@@ -1,5 +1,6 @@
 //! Launches a new renderer window.
 
+#[macro_use] extern crate error_chain;
```

Comment:
Odd, I've been writing mine as

```Rust
#[macro_use]
extern crate error_chain;
```

Is it documented anywhere what style these should be?  I wasn't able to find documentation for this.

Reply:
Note: I'm not requesting this be changed I just want to know if I've been doing mine wrong.

Reply:
I typically have been doing it your way, but there are some merits to the one liner.

Reply:
I'm completely indifferent - I'll change to the amethyst standard :)

Reply:
Did you do `cargo fmt`?